### DISCUSSION TITLE
Octernship

### DISCUSSION THREAD

**Discussion by sshdopey** [community/community#49657] [GitHub Education]
I have applied for the four octernship, but one of the projects automatically assigned by github classroom bot does not even contain a readme file so I am unable to go about that project what should I d to fix this. Am i able to withdraw from that project and apply for a new one?
it is the to-do-list with appwire and flutter

**Comment by sshdopey** [comment]
Please how do i contact the Octernship team. This would be of great help to me.

**Comment by dakshdeepHERE** [comment]
@DestinedCodes I had this issue too and the problem is that I was not able to contact the maintainers too so I found a temp fix for this issue which was to go back to the octernship page and check the project issue from the projects tab itself I will attach the pic too. 
<img width="415" alt="image" src="https://user-images.githubusercontent.com/94860622/225326465-e142144c-4b58-4e27-b269-f2844c370e33.png">
The above project had no read me file in the bot repo but contains all the required info if you click on the project info from the octernship project page.

`Hope this helps you for further assistance you can tag me I will try my best to help you out`
